File: android/lib/model/mitra.dart
import 'package:yunikah/model/asset.dart';

class Mitra {
  int id;
  String name;

  Asset image;

  Mitra({
    this.id,
    this.name,
    this.image
  });

  factory Mitra.parseFromJson(Map<String, dynamic> data) => Mitra(
    id: data['id'],
    name: data['name'],
    image: Asset.parseFromJson(data['image'])
  );
}

List<Mitra> mitraFromJson(List<dynamic> data) => data.map((e) => Mitra.parseFromJson(e)).toList();

File: AndroidStudio/app/src/main/java/com/kosim/yunikah/model/Mitra.kt
package com.kosim.yunikah.model

import com.google.gson.annotations.SerializedName

data class Mitra(
    @SerializedName("id")
    val id : Int?,
    @SerializedName("name")
    val name : String?,
    @SerializedName("keterangan")
    val keterangan : String?,
    @SerializedName("image")
    val image : Asset?
) : Response()